firefox-ios/Client/FeatureFlags/NimbusFlaggableFeature.swift

```diff
@@ -80,7 +80,6 @@ enum NimbusFeatureFlagID: String, CaseIterable {
                 .tabTrayUIExperiments,
                 .toolbarRefactor,
                 .trackingProtectionRefactor,
-                .passwordGenerator,
```

Comment:
thanks for the clean up! Did we also remove it from the Feature Flags spreadsheet?

Reply:
should we remove `passwordGenerator` case from `NimbusFeatureFlagID` as well?

Reply:
> should we remove passwordGenerator case from NimbusFeatureFlagID as well?

We definitely should. Missed it. Thanks 😄 Fixed in https://github.com/mozilla-mobile/firefox-ios/pull/25579/commits/f5379d1f69b391b2c0177a5d00ccec9efbdaa680

Reply:
> thanks for the clean up! Did we also remove it from the Feature Flags spreadsheet?
I marked it as ready to be removed from nimbus and linked this PR in the `removed PR` field 😄

Reply:
thank you!!